app/controllers/kudos_controller.rb

```diff
@@ -2,24 +2,16 @@
 
 class KudosController < ApplicationController
   before_action :set_kudos, only: %i[show edit update destroy]
```

Comment:
You do not have show, edit update and destroy actions anymore, so I would suggest to get rid of this callback :)